e2e/wdio/platform/tests/action-list-item.e2e-spec.ts

```diff
@@ -5,36 +5,41 @@ import { acceptAlert, click, getAttributeByName, getElementArrayLength, refreshP
 
 describe('Action List Item Test Suite:', function() {
     const actionListPg = new ActionListItemPo();
```

Comment:
Maybe it is a good time to get rid of Pg PO and replace it with Page everywhere?

Reply:
you mean to make it actionListPage, correct?